protocol/app/app.go

```diff
@@ -646,13 +646,13 @@ func New(
 			// environments.
 			// app.Server.ExpectBridgeDaemon(daemonservertypes.MaximumAcceptableUpdateDelay(daemonFlags.Bridge.LoopDelayMs))
 			go func() {
-				if err := bridgeclient.Start(
+				app.BridgeClient = bridgeclient.NewClient(logger)
+				if err := app.BridgeClient.Start(
```

Comment:
nit: not related to this PR, can we rename `pricefeedclient.StartNewClient` to `pricefeedclient.Start` to be consistent?

Reply:
Yes, task filed based on comment from another PR!